app_feup/lib/view/Pages/exams_page_view.dart

```diff
@@ -3,6 +3,7 @@ import 'package:uni/model/entities/exam.dart';
 import 'package:flutter/material.dart';
 import 'package:flutter_redux/flutter_redux.dart';
 import 'package:uni/view/Pages/secondary_page_view.dart';
+import 'package:uni/view/Widgets/exam_card.dart';
```

Comment:
Is this an unused import?

Reply:
No, line 123 of exams_page_view.dart uses the bool function isHighlighted defined in that file (exam_card.dart).

Reply:
I see. I don't think this function should be in that file, since it's shared between both places. Can we create `exam` file in the controller and put it there?

Reply:
Yes, of course!